### ISSUE TITLE
AMP doesn't work

### ISSUE THREAD

**Issue by nocwed** [M60TM/TF2-BaseObject-Hook#1]
After i apply "custom dispenser type" "build amplifier" and try to build a dispenser, this error pops up
```
L 08/09/2023 - 22:15:43: [SM] Exception reported: Inflictor index 144 is not valid
L 08/09/2023 - 22:15:43: [SM] Blaming: custom-attr-starter-pack/amp_rewrite.smx
L 08/09/2023 - 22:15:43: [SM] Call stack trace:
L 08/09/2023 - 22:15:43: [SM]   [0] TF2_AddCondition
L 08/09/2023 - 22:15:43: [SM]   [1] Line 31, amp_rewrite.sp::TF2BH_DispenserStartHealing
L 08/09/2023 - 22:15:43: [SM]   [3] Call_Finish
L 08/09/2023 - 22:15:43: [SM]   [4] Line 273, tf2bh/dhooks.sp::DHook_DispenserStartHealingPost
```

**Comment by M60TM** [comment]
After the update(I don't know what update cause this), dispenser cannot be inflictor for add condition.

**Comment by nocwed** [comment]
Aw, what a shame. Thanks for the response anyways!

**Comment by M60TM** [comment]
It was resolved when [54f48e0](https://github.com/M60TM/TF2-BaseObject-Hook/commit/54f48e08c203e196d360f4ee6a42364c0244ce1a). Check out.
